fastlane/Fastfile

```diff
@@ -0,0 +1,83 @@
+# Customise this file, documentation can be found here:
+# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
+# All available actions: https://docs.fastlane.tools/actions
+# can also be listed using the `fastlane actions` command
+# Change the syntax highlighting to Ruby
+# All lines starting with a # are ignored when running `fastlane`
+# If you want to automatically update fastlane if a new version is available:
+# update_fastlane
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "2.120.0"
+default_platform :ios
+# Set default target for get_version_number()
+ENV["FL_VERSION_NUMBER_TARGET"] = "Examples"
+SPACESHIP_SKIP_2FA_UPGRADE=1
+platform :ios do
+  before_all do
+    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
+  end
+  lane :beta do
+    increment_build_number(
+      build_number: latest_testflight_build_number + 1,
+      xcodeproj: Dir.glob("#{ENV['PWD']}/Apps/Examples/Examples.xcodeproj").first
+    )
+    deploy_to_testflight
+  end
+  desc "Submit a new Beta Build to Apple TestFlight"
+  desc "This will also make sure that the signing certificate and provisioning profiles are up to date."
+  lane :build_and_submit do
+    spm(
+      command: "update",
+      configuration: "release"
+    )
```

Comment:
It looks like we don't need `swift package update` call here as there is no Package.swift manifest.